invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html

```diff
@@ -65,6 +65,8 @@
             TU Graz
             <br>
             {{_ ("Repository")}}
+            <br>
+            {{_ ("Library & Archives")}}
```

Comment:
I have only a comment for the commit message header. This is for me not a refactor type. The type refactor indicates that the result of the code is not changed and this is here the case. Therefore it is a feature for me. Further the part in which it is stated that the #110 is fixed should go in the footer of the commit body.

Reply:
good to know.